tests/Shared/Aspire.Templates.Testing.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="$(RepoRoot)src\Shared\Workload.props" />
```

Comment:
Why is this still named "Workload"?

Reply:
This seemed related to the actual workload, but I'm not sure. @joperezr Any idea about this?